share/test_data.sql

```diff
@@ -1,76 +1,138 @@
 PRAGMA foreign_keys = ON;
 -- password: P@ssw0rd
-INSERT INTO 'users' VALUES(1,'john_doe','john_doe','$argon2i$v=19$m=32768,t=3,p=1$Gwe2aqtW9TbCpSosuN0O6Q$ISAlqvQF0LJNjj1KMgkBcw','John Doe','test user from test SQL','john.doe@example.com',CURRENT_TIMESTAMP,NULL,NULL,CURRENT_TIMESTAMP);
-INSERT INTO 'users' VALUES(2,'other','other','other','Other User','other test user from test SQL','other@example.com',CURRENT_TIMESTAMP,NULL,NULL,CURRENT_TIMESTAMP);
-INSERT INTO 'roles_users' VALUES('site_owner',1);
-INSERT INTO 'projects' VALUES(1,'Test','Test','test','Test Project.',1,1,CURRENT_TIMESTAMP,NULL);
-INSERT INTO 'projects' VALUES(2,'Other Project','Other-project','other-project','Other Project.',0,1,CURRENT_TIMESTAMP,NULL);
-INSERT INTO 'projects_users' VALUES(1,1,'owner');
-INSERT INTO 'projects_users' VALUES(1,2,'editor');
-INSERT INTO 'projects_users' VALUES(2,1,'owner');
-INSERT INTO 'shop_sections' VALUES(1,1,'bakery products');
-INSERT INTO 'shop_sections' VALUES(2,1,'milk products');
-INSERT INTO 'shop_sections' VALUES(9,2,'other product');
-INSERT INTO 'articles' VALUES(1,1,   1,NULL,NULL,NULL,'flour','');
-INSERT INTO 'articles' VALUES(2,1,   1,NULL,NULL,NULL,'salt','');
-INSERT INTO 'articles' VALUES(3,1,NULL,NULL,NULL,NULL,'water','');
-INSERT INTO 'articles' VALUES(4,1,   2,NULL,NULL,NULL,'cheese','');
-INSERT INTO 'articles' VALUES(5,1,NULL,NULL,NULL,NULL,'love',''); -- has no unit
-INSERT INTO 'articles' VALUES(6,2,   9,NULL,NULL,NULL,'other article','');
-INSERT INTO 'meals' VALUES(1,1,'2000-01-01','breakfast','Best meal of the day!');
-INSERT INTO 'meals' VALUES(2,1,'2000-01-02','lunch','');
-INSERT INTO 'meals' VALUES(3,1,'2000-01-03','dinner','');
-INSERT INTO 'meals' VALUES(9,2,'2000-01-01','other meal','');
-INSERT INTO 'quantities' VALUES(1,1,'Mass',NULL);
-INSERT INTO 'quantities' VALUES(2,1,'Volume',NULL);
-INSERT INTO 'units' VALUES(1,1,1,0.001,0,'g','grams');
-INSERT INTO 'units' VALUES(2,1,1,1.0,0,'kg','kilograms');
-INSERT INTO 'units' VALUES(3,1,2,1.0,0,'l','liters');
-INSERT INTO 'units' VALUES(4,1,1,1000,0,'t','tons');
-INSERT INTO 'units' VALUES(5,1,1,NULL,0,'p','pinch'); -- no conversion, (in German: Prise)
+INSERT INTO 'users'
+(id,       name,    name_fc,                                                                 password_hash, display_name,                   admin_comment,                  email,    email_verified, token_hash, token_expires,           created) VALUES
+( 1, 'john_doe', 'john_doe', '$argon2i$v=19$m=32768,t=3,p=1$Gwe2aqtW9TbCpSosuN0O6Q$ISAlqvQF0LJNjj1KMgkBcw',   'John Doe',       'test user from test SQL', 'john.doe@example.com', CURRENT_TIMESTAMP,       NULL,          NULL, CURRENT_TIMESTAMP),
```

Comment:
I thought about moving the long `password_hash` of `john_doe` out of this line and setting it below using an `UPDATE` query for readability of this block. What do you think?